### ISSUE TITLE
Add command for website launch

### ISSUE THREAD

**Issue by brettsmason** [eighteen73/wp-cli-tools#11]
We have recently had issues with a website launch due to https://core.trac.wordpress.org/ticket/59111
The suggestion was made to include a new command, something like `wp eighteen73 launch` to deal with all the various operations that need to be acomplished during a website launch, including but not limited to:

```
wp search-replace "//old-domain.com" "//new-domain.com"
wp yoast index --reindex
wp transient delete --all
```

**Comment by edjeavons** [comment]
Good idea.

How about including the domains in the command like this, and it'll skip the search-replace step if they're omitted?
```shell
wp eighteen73 launch --old-domain=beta.example.com,example.test --new-domain=example.com
```
That would allow multiple domains to be entered in situations where >1 have found their way into the DB for some reason (not uncommon in my experience).

Or would we prefer to be prompted for the old & new domains interactively.

**Comment by edjeavons** [comment]
Available in the latest release v0.4.0
